### DISCUSSION TITLE
Welcome to phoenix-rtos-project Discussions!

### DISCUSSION THREAD

**Discussion by nalajcie** [phoenix-rtos/phoenix-rtos-project#625] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KenDickey** [comment]
Greetings,

I am an old hobby programmer, retired 20 years ago from commercial software development but still doing open source SW.

I have been playing mostly with Cuis Smalltalk and have ported the OpenSmalltalk VM to aarch64/arm64 and riscv64, running on Linux under XWindows/Wayland or via FrameBuffer+libevdev.  Smalltalk is portable, which means that I can be working on my wife's Macintosh, copy the working image to my Raspberry Pi 4 (aarch64) or VisionFive2 (riscv64) and continue where I left off, with the windows in the same places -- pixel per pixel.

I have seen Minix, Genode, et al and am interested in the promise of microkernels to be robust in the face of buggy device drivers.

I like software to be comprehensible and look for projects which are large enough to be interesting but small enough to be tractable. 

So I would like to be able to run Phoenix-RTOS on a VisionFive2, get the framebuffer up, and run the Cuis Smalltalk IDE.  That's my goal here.

Thanks much for making Phoenix-RTOS open source!
-KenD    [Ken (dot) Dickey (at) Whidbey (dot) COM]
